tests/sles4sap/qesapdeployment/deploy.pm

```diff
@@ -26,7 +26,9 @@ sub run {
     {
         if (qesap_file_find_string(file => $ret[1], search_string => 'Missing sudo password')) {
             record_info('DETECTED ANSIBLE MISSING SUDO PASSWORD ERROR');
-            @ret = qesap_execute(cmd => 'ansible', cmd_options => '--profile', verbose => 1, timeout => 3600);
+            @ret = qesap_execute(cmd => 'ansible',
```

Comment:
I decided to remove `cmd_options => '--profile', verbose => 1` as the retry does not need them